### PR TITLE
Rachana_Assignment_8_Done

### PR DESCRIPTION
**Pull request by RachanaGJun20**

### CODE REVIEW

File: src/rachanaGJun20/BankingSystem.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+public class BankingSystem {
+
+	int debitcounts;
+	int creditcounts;
+	int printbalancecounts;
+
+	static int totaldebitcounts = 0;
+	static int totalcreditcounts = 0;
+	static int totalprintbalcounts = 0;
```
Comment (KrishnaKTechnocredits) [inline src/rachanaGJun20/BankingSystem.java]: Remove the initialization of all instance variables.

File: src/rachanaGJun20/BankingSystem.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+	void debitAmount() {
+		debitcounts++;
+		totaldebitcounts++;
+	}
+
+	void creditAmount() {
```
Comment (KrishnaKTechnocredits) [inline src/rachanaGJun20/BankingSystem.java]: You forgot to perform an operation in debitAcc() and credit() METHODS. You should have a balance instance variable. You should write the condition to verify the insufficient balance.

File: src/rachanaGJun20/BankingSystem.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	public static void main(String[] args) {
+
+		BankingSystem bankingsystem1 = new BankingSystem();
+		BankingSystem bankingsystem2 = new BankingSystem();
+
+		// Banking for user Amit
+		bankingsystem1.debitAmount();
+		bankingsystem1.debitAmount();
+		for (int j = 0; j <= 3; j++) {
+			bankingsystem1.creditAmount();
```
Comment (KrishnaKTechnocredits) [inline src/rachanaGJun20/BankingSystem.java]: creditAmount() should have one parameter that will add in main balance.

File: src/rachanaGJun20/BankingSystem.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+		}
+		bankingsystem1.printBalance();
+		bankingsystem1.individualTransactionSummary("Amit");
+
+		// Banking for user Sarika
+		for (int i = 1; i <= 5; i++) {
```
Comment (KrishnaKTechnocredits) [inline src/rachanaGJun20/BankingSystem.java]: Remove from here as well.

**Comment by KrishnaKTechnocredits** [comment]
@Rachna you should replied on comment.

**Comment by RachanaGJun20** [comment]
Okay Krishna

**Pull request merged**